{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -51,3 +52,9 @@ docs: ## Build docs using Sphinx and copy to docs folder (this makes it easy to
 
 ipython: ## Provides an interactive ipython prompt
 	docker exec -it $(CONTAINER_NAME) ipython
+
+setup-dvc: ## Initializes DVC in project repo
+	docker exec -it $(CONTAINER_NAME) dvc init
+
+setup-dvc-remote: ## Adds DVC remote to repo.
```

Comment:
I was following the discussion somewhat -- but here's my question seeing this in the final state -- what happens if I run this again from the makefile? Does it setup another remote? Or does it just fail?

Reply:
I could see an argument against having this here. We don't have a make command to create a remote git repo. That is explicitly done using git commands. We have some pointers on how to do it in the README... but not in the MakeFile. I think we should be consistent -- if we choose to do this, we should add a think for initializing a git repo too. Else, remove this and put the command in the README for people to follow and do explicitly (not using the Makefile).  Carrying this argument all the way through -- this probably means we should not have the DVC remote URL as a cookiecutter parameter either.... What do you all think?

Reply:
> I was following the discussion somewhat -- but here's my question seeing this in the final state -- what happens if I run this again from the makefile? Does it setup another remote? Or does it just fail?

It'll just fail because that specific remote has already been defined.

Reply:
> I could see an argument against having this here. We don't have a make command to create a remote git repo. That is explicitly done using git commands. We have some pointers on how to do it in the README... but not in the MakeFile. I think we should be consistent -- if we choose to do this, we should add a think for initializing a git repo too. Else, remove this and put the command in the README for people to follow and do explicitly (not using the Makefile). Carrying this argument all the way through -- this probably means we should not have the DVC remote URL as a cookiecutter parameter either.... What do you all think?

I like this. I think you're right about adding DVC commands to Makefiles being inconsistent with how we use git. Since it's not guaranteed someone will use DVC anyways we could remove the remote URL cookiecutter parameter and just have documentation in the README<